src/components/ChatEntry.js

```diff
@@ -2,21 +2,43 @@ import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
 
+const diffTime = (time) => {
+  const timeMS = Date.now() - Date.parse(time);
+  const timeDiff = new Date(timeMS);
+  return Math.abs(timeDiff.getUTCFullYear() - 1970);
+};
+
 const ChatEntry = (props) => {
+  const heart = props.liked ? '❤️' : '🤍'
   return (
     <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+      <h2 className="entry-name">{props.sender}</h2>
       <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+        <p>{props.body}</p>
+        <p className="entry-time">{diffTime(props.timeStamp)} years ago</p>
+        <button
+          className="like"
+          onClick={() => {
+            props.flipLiked(props.id);
+            props.increaseLikes();
+          }}
```

Comment:
I like that you used an anonymous function here.  🙂

(Anonymous functions don't get the love they deserve!)